### PR TITLE
Add socratic 104

### PR DESCRIPTION
**Pull request by jnewbery**
I just set the date to 09-11 (one month after previous socratic). Obviously update the filename before publishing to get the right date.

### CODE REVIEW

**Comment by jnewbery** [comment]
Updated with September links for Bitcoin Core, mailing lists, etc

@btsea @andrerfneves over to you :slightly_smiling_face:

File: _posts/2021-09-11-socratic-seminar-104.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+- [Bitcoin Core v0.21.2 release candidate](https://bitcoincore.org/bin/bitcoin-core-0.21.2/)
+- [Bitcoin Core v22.0](https://bitcoincore.org/bin/bitcoin-core-22.0/)
+- [c-lightning v0.10.1](https://github.com/ElementsProject/lightning/releases/tag/v0.10.1)
+- [rust-lightning v0.0.100](https://github.com/rust-bitcoin/rust-lightning/releases/tag/v0.0.100)
+
+## Events and Podcasts
+
+
+## Mining
+
```
Comment (Xekyo) [inline _posts/2021-09-11-socratic-seminar-104.md:127]:
```suggestion
- [Blockstream acquires Spondoolies, raises $210M](https://twitter.com/CoinDesk/status/1430131880708526089)

```

File: _posts/2021-09-11-socratic-seminar-104.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+- [Decide which coin selection solution to use based on waste metric](https://github.com/bitcoin/bitcoin/pull/22009)
+
+### BIPS
+
+
+### lnd
+
+
+### BOLTs
+
```
Comment (Xekyo) [inline _posts/2021-09-11-socratic-seminar-104.md:108]:
```suggestion
- [Bolt12 Offers (updated with C-Lightning 0.10.1)](http://bolt12.org/)

```

File: _posts/2021-09-11-socratic-seminar-104.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+- [Newsletter #165](https://bitcoinops.org/en/newsletters/2021/09/08/)
+- [Newsletter #166](https://bitcoinops.org/en/newsletters/2021/09/15/)
+- [Newsletter #167](https://bitcoinops.org/en/newsletters/2021/09/22/)
+
+## Network Data
+
+
+### Research
+
+- [eltoo with Anyprevout and Taproot](https://yakshaver.org/2021/07/26/first.html)
```
Comment (Xekyo) [inline _posts/2021-09-11-socratic-seminar-104.md:84]:
```suggestion
- [eltoo with Anyprevout and Taproot](https://yakshaver.org/2021/07/26/first.html)
- [Simplicity support in Elements](https://twitter.com/n1ckler/status/1422587944838303745)
```

File: _posts/2021-09-11-socratic-seminar-104.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+- [Newsletter #160](https://bitcoinops.org/en/newsletters/2021/08/04/)
+- [Newsletter #161](https://bitcoinops.org/en/newsletters/2021/08/11/)
+- [Newsletter #162](https://bitcoinops.org/en/newsletters/2021/08/18/)
+- [Newsletter #163](https://bitcoinops.org/en/newsletters/2021/08/25/)
+- [Newsletter #164](https://bitcoinops.org/en/newsletters/2021/09/01/)
+- [Newsletter #165](https://bitcoinops.org/en/newsletters/2021/09/08/)
+- [Newsletter #166](https://bitcoinops.org/en/newsletters/2021/09/15/)
+- [Newsletter #167](https://bitcoinops.org/en/newsletters/2021/09/22/)
+
+## Network Data
```
Comment (Xekyo) [inline _posts/2021-09-11-socratic-seminar-104.md:79]:
```suggestion
## Network Data
- [Lightning network capacity](https://bitcoinvisuals.com/ln-capacity)
- [Hashrate recovers to 90% of ATH](https://miningpoolstats.stream/bitcoin)
- [LN routing traffic growing](https://twitter.com/LNMarkets/status/1433718559306362880)
- [Taproot readiness: 55% of nodes upgraded](https://twitter.com/taproot_signal/status/1442851595109257216)
```

**Comment by Xekyo** [comment]
I couldn't push to jnewbery's repo, so I added some links as review suggestions.

**Comment by andrerfneves** [comment]
@btsea @jnewbery @Xekyo LN links pushed

**Comment by jnewbery** [comment]
@Xekyo I've added you to the github org, so I think you should be able to push to PR branches now.

**Comment by Xekyo** [comment]
Thanks @jnewbery!

**Comment by btsea** [comment]
@Xekyo @jnewbery @andrerfneves thank you guys!

File: _posts/2021-09-11-socratic-seminar-104.md [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+- [Newsletter #165](https://bitcoinops.org/en/newsletters/2021/09/08/)
+- [Newsletter #166](https://bitcoinops.org/en/newsletters/2021/09/15/)
+- [Newsletter #167](https://bitcoinops.org/en/newsletters/2021/09/22/)
+- [Newsletter #168](https://bitcoinops.org/en/newsletters/2021/09/29/)
+
+## Network Data
+
+- [Average vsize of inputs has diminished 14.4% YTD](https://twitter.com/murchandamus/status/1438521484910075919)
+- [bech32m adoption (wallets, explorers, exchanges)](https://en.bitcoin.it/wiki/Bech32_adoption)
+- [Bitfinex sends accidental $23.7 fee](https://www.theblockcrypto.com/amp/post/118753/bitfinex-just-spent-23-7-million-in-fees-to-make-one-ethereum-transaction)
```
Comment (Xekyo) [inline _posts/2021-09-11-socratic-seminar-104.md:99]:
```suggestion
- [Bitfinex sends accidental $23.7M fee](https://www.theblockcrypto.com/amp/post/118753/bitfinex-just-spent-23-7-million-in-fees-to-make-one-ethereum-transaction)
- ```

File: _posts/2021-09-11-socratic-seminar-104.md [hunk 6]
```diff
@@ -0,0 +1,304 @@
...
+-  [NSO Group iMessage Zero-Click Exploit Captured in the Wild](https://citizenlab.ca/2021/09/forcedentry-nso-group-imessage-zero-click-exploit-captured-in-the-wild/)
+- [Punk Protocol drained of $8.9m, ~$5m returned by front-running bot](https://rekt.news/punkprotocol-rekt/)
+- [Secure env vars of *all* public Travis CI
+ repositories were injected into PR builds](https://travis-ci.community/t/security-bulletin/12081), [context](https://mobile.twitter.com/peter_szilagyi/status/1437646118700175360)
+- [Solana DDoS](https://solana.com/news/9-14-network-outage-initial-overview/)
+- [VEE Finance drained of $34m](https://veefi.medium.com/the-main-cause-of-vee-finance-attack-7a8475085ec5)
+- [XToken drained of $4.5m](https://medium.com/xtoken/xsnx-post-mortem-666d35071f38)
+- [$90 Million stolen from Liquid exchange](https://twitter.com/Liquid_Global/status/1428176357515612165?s=20), [funds flowing through tornado.cash, wasabi and defi protocols](https://twitter.com/ErgoBTC/status/1430685397659566080), [more info](https://twitter.com/ErgoBTC/status/1431026764361437190)
```
Comment (Xekyo) [inline _posts/2021-09-11-socratic-seminar-104.md:168]:
```suggestion
- [$90 Million stolen from Liquid exchange](https://twitter.com/Liquid_Global/status/1428176357515612165), [funds flowing through tornado.cash, wasabi and defi protocols](https://twitter.com/ErgoBTC/status/1430685397659566080), [more info](https://twitter.com/ErgoBTC/status/1431026764361437190)
```

File: _posts/2021-09-11-socratic-seminar-104.md [hunk 7]
```diff
@@ -0,0 +1,304 @@
...
+- [TSMC increasing wafer prices](https://www.theblockcrypto.com/linked/115708/bitmain-bitcoin-miner-wafer-price-increase)
+
+## Miscellaneous
+
+- [Accounts, Strict Access Lists, and UTXOs](https://forum.celestia.org/t/accounts-strict-access-lists-and-utxos/37)
+- [ACINQ Strike launched on mainnet](https://medium.com/@ACINQ/strike-our-stripe-like-api-for-lightning-is-live-cd1dce76ce2e)
+- [BitcoinBinary - tracking reproducibility of hardware signer firmwares and wallet software](https://bitcoinbinary.org/)
+- [Bitcoin Core 22.0 rc testing guide](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/22.0-Release-Candidate-Testing-Guide)
+- [BitGo migrating to native segwit change outputs](https://blog.bitgo.com/native-segwit-change-outputs-for-bitcoin-c021406aaae2?gi=f925a1735539)
+- [BitMEEX anonunces developer grant fo Sjors Provoost](https://blog.bitmex.com/new-open-source-bitcoin-developer-grant-for-sjors-provoost-2/)
```
Comment (Xekyo) [inline _posts/2021-09-11-socratic-seminar-104.md:275]:
```suggestion
- [BitMEX anonunces developer grant fo Sjors Provoost](https://blog.bitmex.com/new-open-source-bitcoin-developer-grant-for-sjors-provoost-2/)
```

File: _posts/2021-09-11-socratic-seminar-104.md [hunk 8]
```diff
@@ -0,0 +1,304 @@
...
+- [Committee-driven MEV smoothing](https://ethresear.ch/t/committee-driven-mev-smoothing/10408)
+- [Compass commits to sponsor Bitcoin Core developer Jon Atack](https://compassmining.io/education/compass-core-developer-donation/)
+- [eltoo with Anyprevout and Taproot](https://yakshaver.org/2021/07/26/first.html)
```
Comment (Xekyo) [inline _posts/2021-09-11-socratic-seminar-104.md:278]: This was mentioned above already.

**Review by Xekyo — COMMENTED** [review]
Just a few typos

**Pull request merged**